src/test/java/com/uber/jenkins/phabricator/utils/TestUtils.java

```diff
@@ -83,6 +104,15 @@ public static CoverageResult getDefaultCoverageResult() {
         return getCoverageResult(100.0f, 100.0f, 100.0f, 100.0f, 100.0f);
     }
 
+    public static JSONObject getJSONFromFile(Class<?> klass, String filename) throws IOException {
```

Comment:
not sure if this `Class<?>` is the right thing, or even what that means (how do java?) but this works